modules/deeparg/predict/main.nf

```diff
@@ -0,0 +1,40 @@
+def VERSION="1.0.2"
+
+process DEEPARG_PREDICT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::deeparg=1.0.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity//deeparg:1.0.2--pyhdfd78af_1' :
+        'quay.io/biocontainers/deeparg:1.0.2--pyhdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(fasta), val(model)
+    tuple path(db)
```

Comment:
not sure if the tuples here are correct?

Reply:
Oh no, why?

Reply:
A tuple with just one element (`path(db)`) seems to be a bit of an overkill, no?

Reply:
Ah crap yeah! I'll fix that tomorrow

Reply:
You can have that review ;)